onnxscript/ir/_core.py

```diff
@@ -1055,6 +1056,18 @@ def _quoted(string: str) -> str:
     return f'"{string}"'
 
 
+class Usage(NamedTuple):
+    """A usage of a value in a node.
+
+    Attributes:
+        node: The node that uses the value.
+        idx: The input index of the value in the node.
+    """
+
+    node: Node
+    idx: int
```

Comment:
nit: why not `index`? Seems short enough

Reply:
mypy was complaining that I was overwriting a builtin method. I guess that's a method in tuple

Reply:
Yeah this - https://www.w3schools.com/python/ref_tuple_index.asp